komga-webui/src/locales/en.json

```diff
@@ -279,65 +279,65 @@
+      "dialog_title": "Delete collection",
+      "dialog_title_multiple": "Delete collections",
+      "warning_html": "Remove the {name} collection from this server permanently? (Your media files will not be affected.)",
+      "warning_multiple_html": "Remove these {count} collections from this server permanently? (Your media files will not be affected.)"
     },
     "delete_library": {
       "button_confirm": "Delete",
-      "confirm_delete": "Yes, delete the library \"{name}\"",
-      "title": "Delete Library",
-      "warning_html": "The library <b>{name}</b> will be removed from this server. Your media files will not be affected. This <b>cannot</b> be undone. Continue?"
+      "confirm_delete": "Yes, delete the \"{name}\" library",
+      "title": "Delete library",
+      "warning_html": "Remove the {name} library from this server permanently? (Your media files will not be affected.)"
     },
     "delete_readlist": {
       "button_confirm": "Delete",
-      "confirm_delete": "Yes, delete the read list \"{name}\"",
-      "confirm_delete_multiple": "Yes, delete {count} read lists",
-      "dialog_title": "Delete Read List",
-      "dialog_title_multiple": "Delete Read Lists",
-      "warning_html": "The read list <b>{name}</b> will be removed from this server. Your media files will not be affected. This <b>cannot</b> be undone. Continue?",
-      "warning_multiple_html": "{count} read lists will be removed from this server. Your media files will not be affected. This <b>cannot</b> be undone. Continue?"
+      "confirm_delete": "Yes, delete the reading list \"{name}\"",
+      "confirm_delete_multiple": "Yes, delete {count} reading lists",
+      "dialog_title": "Delete reading list",
+      "dialog_title_multiple": "Delete reading lists",
+      "warning_html": "Remove the \"{name}\" reading list from this server permanently? (Your media files will not be affected.)",
+      "warning_multiple_html": "Remove {count} reading lists from this server permanently? (Your media files will not be affected.)"
     },
     "delete_series": {
       "button_confirm": "Delete",
       "confirm_delete": "Yes, delete series \"{name}\" and its files",
       "confirm_delete_multiple": "Yes, delete {count} series and their files",
-      "dialog_title": "Delete Series",
-      "warning_html": "The Series <b>{name}</b> will be removed from this server alongside with stored media files. This <b>cannot</b> be undone. Continue?",
-      "warning_multiple_html": "{count} series will be removed from this server alongside with stored media files. This <b>cannot</b> be undone. Continue?"
+      "dialog_title": "Delete series",
+      "warning_html": "Remove the {name} series from this server along with its stored media files permanently?",
+      "warning_multiple_html": "Remove these {count} series from this server along with its stored media files permanently?"
```

Comment:
You are removing the **big fat warning** that files will be removed, and **cannot be undone**.